Makefile.tools

```diff
@@ -189,11 +188,10 @@ $(SHELLCHECK) shellcheck: $(TOOLS_DIR)
 		[[ -f $(SHELLCHECK) ]] && exit 0 ;\
 		cd $$(mktemp -d) ;\
 		OS=$(shell go env GOOS) && ARCH=$(shell uname -m) && \
-		curl -sSLo shellcheck-stable.tar.xz https://github.com/koalaman/shellcheck/releases/download/stable/shellcheck-stable.$${OS}.$${ARCH}.tar.xz ;\
-		tar -xJf shellcheck-stable.tar.xz ;\
-		cp shellcheck-stable/shellcheck $(SHELLCHECK) ;\
-		version=$(SHELLCHECK_VERSION) ;\
```

Comment:
why remove the version checks?

Reply:
With this patch, the exact version is specified in the URL, so we can be sure the same version is fetched.